plugins/catalog-backend/src/service/request/entitiesBatchRequest.ts

```diff
@@ -21,6 +21,7 @@ import { z } from 'zod';
 const schema = z.object({
   entityRefs: z.array(z.string()),
   fields: z.array(z.string()).optional(),
+  filter: z.array(z.string()).optional(),
```

Comment:
this will never happen though? it's in the query, not the body

Reply:
True, will remove!